README.md

```diff
@@ -6,18 +6,60 @@
 [![NSF-2126413](https://img.shields.io/badge/NSF-2126413-blue.svg)](https://nsf.gov/awardsearch/showAward?AWD_ID=2126413)
 [![Hits](https://hits.seeyoufarm.com/api/count/incr/badge.svg?url=https%3A%2F%2Fgithub.com%2FCloud-Drift%2Fclouddrift&count_bg=%2368C563&title_bg=%23555555&icon=&icon_color=%23E7E7E7&title=hits&edge_flat=false)](https://hits.seeyoufarm.com)
 
-The *CloudDrift* project aims at accelerating the use of Lagrangian data for atmospheric, oceanic, and climate sciences. It is currently funded by the [NSF EarthCube program](https://www.earthcube.org/info) through [EarthCube Capabilities Grant No. 2126413](https://www.nsf.gov/awardsearch/showAward?AWD_ID=2126413). The lead Principal Investigator of the project is [Shane Elipot](https://github.com/selipot) and its development is led by [Philippe Miron](https://github.com/philippemiron).
+CloudDrift accelerates the use of Lagrangian data for atmospheric, oceanic, and climate sciences.
+It is funded by [NSF EarthCube](https://www.earthcube.org/info) through the
+[EarthCube Capabilities Grant No. 2126413](https://www.nsf.gov/awardsearch/showAward?AWD_ID=2126413).
 
-This project is generating the *clouddrift* library as an open source software written in python. This software requires a number of modules that need to exist within your working python environment. The list of necessary modules is contained in the YAML file `environment.yml`. This file can be used with a package manager such as conda to create the necessary python environment. From the command line, run
+## Getting started
 
-`conda env create --file environment.yml`
+### Install CloudDrift
 
-Once the library is in a *beta* stage, a conda/pip packages will be available to simplify the installation process.
+#### pip
 
-This repository is organized as follows:
-- `clouddrift/`: modules of the *clouddrift* library.
-- `data/`: processing scripts for various Lagrangian datasets, including the GDP historical dataset.
-- `docs/`: documentation *in-progress* available at [cloud-drift.github.io/clouddrift](https://cloud-drift.github.io/clouddrift/).
-- `examples/`: series of Jupyter Notebooks showcasing the library use cases.
-- `tests/`: test-suite.
-- `dev/`: development Jupyter Notebooks use to brainstorm new ideas and discussions.
+You can install CloudDrift using pip via PyPI:
+
+```
+pip install clouddrift
+```
+
+If you need the latest development version, get it from GitHub:
+
+```
+pip install git+https://github.com/cloud-drift/clouddrift
+```
+
+#### Conda
+
+You can also install CloudDrift using Conda:
+
+```
+git clone https://github.com/cloud-drift/clouddrift
+cd clouddrift
+conda env create --file environment.yml
+```
+
+Once the library is in Beta, a conda-forge package will be available to simplify the installation process.
```

Comment:
I think the library is beyond *Beta* at this stage, isn't it? I recall the issue of not having a conda install yet is because of the the awkward library dependency. What about writing:

"In the future, a conda-forge package will be available to simplify the installation process."

Reply:
I agree, no need to label it as Beta IMO, but wasn't sure so I left it alone.

Reply:
Sounds good! I see that the awkward version is now updated on conda (https://anaconda.org/conda-forge/awkward). I'll give it a try on Monday, but it should work now to update the environment and remove installing packages with pip.

Reply:
That's great news!

Reply:
I am getting confused here again: now that `awkward` can be installed with conda, we can strip the `environment.yml` file of `pip` and pip install of awkward and move the awkward install back to conda ...  but should we still specify to install clouddrift?